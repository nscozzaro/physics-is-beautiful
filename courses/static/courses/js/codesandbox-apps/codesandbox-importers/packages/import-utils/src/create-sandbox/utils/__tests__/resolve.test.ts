import { getDirectoryPaths } from "../resolve";

describe("resolve", () => {
  describe("getDirectoryPaths", () => {
    it("can resolve dir paths", () => {
      const existingDirs = [
        {
          directoryShortid: undefined,
          title: "world",
          shortid: "dir1",
        },
        {
          directoryShortid: "dir1",
          title: "hello",
          shortid: "dir2",
        },
      ];

      expect(getDirectoryPaths(existingDirs)).toMatchSnapshot();
    });
  });
});
