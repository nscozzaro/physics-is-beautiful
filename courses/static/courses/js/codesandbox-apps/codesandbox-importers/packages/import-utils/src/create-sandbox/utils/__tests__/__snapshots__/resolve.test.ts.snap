// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resolve getDirectoryPaths can resolve dir paths 1`] = `
Object {
  "/world": Object {
    "directoryShortid": undefined,
    "shortid": "dir1",
    "title": "world",
  },
  "/world/hello": Object {
    "directoryShortid": "dir1",
    "shortid": "dir2",
    "title": "hello",
  },
}
`;
